class-jetpack-currencies.php

```diff
@@ -0,0 +1,175 @@
+<?php
+/**
+ * Jetpack_Currencies: Utils for displaying and managing currencies.
+ *
+ * @package    Jetpack
+ * @since      9.1.0
+ */
+
+/**
+ * General Gutenberg editor specific functionality
+ */
+class Jetpack_Currencies {
+	/**
+	 * Currencies definition
+	 */
+	const CURRENCIES = array(
```

Comment:
For folks reviewing, this has been moved from `Jetpack_Simple_Payments`: https://github.com/Automattic/jetpack/pull/17381/files#diff-b73125af4e0fe93b3939f29ff9c7e6fc33d17e564a32fb7ea8d082d1009e3401